polylogue/branch_explorer.py

```
@@ -621,6 +648,37 @@ def build_branch_html(conversation: BranchConversationSummary, *, theme: str = "
+            branch_lines = branch_path.read_text(encoding="utf-8").splitlines()
+            from difflib import unified_diff
+
+            diff_iter = unified_diff(
+                canonical_lines,
+                branch_lines,
+                fromfile=canonical_path.name,
+                tofile=branch_path.name,
+                lineterm="",
+            )
+            lines = list(islice(diff_iter, max_lines + 1))
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)